wiki/BanchoBot/th.md

```diff
@@ -0,0 +1,177 @@
+
+หากระบุผู้ใช้ที่ไม่เคยเล่น osu! เลย BanchoBot จะตอบกลับว่า `User not found` ถึงแม้ว่าจะมีชื่อผู้ใช้นั้นในระบบก็ตาม
+
+ตัวอย่างการใช้คำสั่ง
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+คำสั่ง `!stats` จะสามารถแสดงสถานะได้ 7 สถานะ ดังนี้ `Editing` *(กำลังแก้ไขบีทแมพ)*, `Idle` *(ไม่ได้ทำอะไร)*, `Lobby` *(อยู่หน้าล็อบบี้ของเกมผู้เล่นหลายคน)*, `Modding` *(กำลังวิจารณ์บีทแมพ)*, `Multiplayer` *(อยู่ในห้องผู้เล่นหลายคน)*, `Multiplaying` *(กำลังเล่นอยู่ในห้องผู้เล่นหลายคน)* และ `Playing` *(กำลังเล่น)*
+
+ตัวอย่างการใช้คำสั่ง
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy is Playing:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+### Where
+
+```
+!where <ชื่อผู้ใช้>
+```
+
+คำสั่ง `!where` จะแสดงสถานที่ของผู้ใช้ ณ ขณะนั้น โดยทั่วไปจะแสดงเพียงประเทศที่อยู่อาศัย แต่ถ้าผู้ใช้นั้นเปิด `แสดงเมื่องที่คุณอยู่` (`Share your city location with others`) จะแสดงข้อมูลของจังหวัด/รัฐ/เมือง ด้วย
+
+ตัวอย่างการใช้คำสั่ง
+
+```
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
+```
+
+### FAQ
+
+```
+!faq <ข้อความ>
+```
+
+```
+!faq list
+```
+
+คำสั่ง `!faq` จะแสดงข้อมูลของข้อความนั้น แต่ถ้าใช้คำสั่ง `!faq list` จะแสดงข้อความที่สามารถใส่ได้ทั้งหมดแทน โดยที่วไป BanchoBot จะตอบกลับเป็นภาษาอังกฤษ แต่ถ้าเติม[รหัสภาษา 2 ตัว](/wiki/Article_Styling_Criteria#locales)หน้าข้อความตามด้วยเครื่องหมายทวิภาค (`:`) BanchoBot จะตอบกลับเป็นภาษานั้นได้ ถ้าหากมีข้อมูลภาษานั้น
+
+ตัวอย่างการใช้คำสั่ง
+
+```
+13:03 pippi: !faq peppy
+13:03 BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+```
+
+```
+13.04 pippi: !faq th:peppy
+13.04 BanchoBot: คุณ peppy คือผู้ที่อยู่เบื้องหลังและผลักดันเกม osu! จนถึงทุกวันนี้ เขาจะมีสีชื่อสีฟ้าในแชท
+13.04 BanchoBot: ถ้าคุณเห็นเขาอยู่ก็อาจลองทักทายเขาก็ได้แต่อย่ารบกวนเขามากนะ :)
+```
+
+### Report
+
+*สำหรับข้อมูลว่าสิ่งใดควรรายงาน ดูที่[การรายงานพฤติกรรม](/wiki/Reporting_Bad_Behaviour).*
+
+```
+!report <ชื่อผู้ใช้> <เหตุผล>
+```
+
+คำสั่ง `!report` จะส่งข้อมูลการรายงานไปยัง[ทีมผู้ดูแล](/wiki/Global_Moderation_Team) หากชื่อผู้ใช้ประกอบด้วยเครื่องหมายเว้นวรรค ให้แทนที่เครื่องหมายเว้นวรรคด้วยเครื่องหมายสัญประกาศ (`_`) แทน เช่น ผู้ใช้ชื่อ `really cool username` ให้เปลี่ยนเป็นชื่อ `really_cool_username` แทน
+
+สำหรับการรายงานผู้ดูแล ให้ส่งอีเมลไปที่ [support@ppy.sh](mailto:support@ppy.sh)
+
+ตัวอย่างการใช้คำสั่ง
+
+```
+13:10 pippi: !report flyte spamming in #japanese
+13:10 BanchoBot: Chat moderators have been alerted. Thanks for your help.
+```
+
+## เพิ่มเติม
+
+- หน้าผู้ใช้ของ BanchoBot จะแสดงคำว่า "Here since the beginning" แทนที่วันที่เข้าร่วม
```

Comment:
```suggestion
- หน้าผู้ใช้ของ BanchoBot จะแสดงคำว่า "อยู่ตั้งแต่โอสุเริ่มต้น" แทนที่วันที่เข้าร่วม
```
![image](https://user-images.githubusercontent.com/20501245/104838785-b9a5a200-58ef-11eb-980c-b8428eace0bf.png)

Not sure if we should change this

Reply:
I didn't notice that the user page is translated too 😅.